INSTALL.rst.txt

```diff
@@ -0,0 +1,149 @@
+Building and installing NumPy
++++++++++++++++++++++++++++++
+
+**IMPORTANT**: the below notes are about building Numpy, which for most users
+is *not* the recommended way to install Numpy.  Instead, use either a complete
+scientific Python distribution (recommended) or a binary installer - see
+http://scipy.org/install.html.
+
+
+.. Contents::
+
+Prerequisites
+=============
+
+Building NumPy requires the following software installed:
+
+1) For Python 2, Python__ 2.6.x or newer.
+   For Python 3, Python__ 3.2.x or newer.
+
+   On Debian and derivative (Ubuntu): python python-dev
+
+   On Windows: the official python installer on Python__ is enough
+
+   Make sure that the Python package distutils is installed before
+   continuing. For example, in Debian GNU/Linux, distutils is included
+   in the python-dev package.
+
+   Python must also be compiled with the zlib module enabled.
+
+2) Cython >= 0.19 (for development versions of numpy, not for released
+                   versions)
+3) nose__ (optional) 1.0 or later
+
+   This is required for testing numpy, but not for using it.
+
+Python__ http://www.python.org
+nose__ http://somethingaboutorange.com/mrl/projects/nose/
+
+
+.. note:: 
+
+   If you want to build Numpy in order to work on Numpy itself, use
+   ``runtests.py``.  For more details, see
+   http://docs.scipy.org/doc/numpy-dev/dev/development_environment.html
+
+.. note::
+
+   More extensive information on building Numpy (and Scipy) is maintained at
+   http://scipy.org/scipylib/building/index.html
+
+
+Basic Installation
+==================
+
+To install numpy run::
+
+    python setup.py build -j 4 install --prefix $HOME/.local
+
+This will compile numpy on 4 CPUs and install it into the specified prefix.
+To perform an inplace build that can be run from the source folder run::
+
+    python setup.py build_ext --inplace -j 4
+
+The number of build jobs can also be specified via the environment variable
+NPY_NUM_BUILD_JOBS.
+
+
+Choosing compilers
+==================
+
+On OS X and Linux, all common compilers will work.  Note that for Fortran,
+``gfortran`` is strongly preferred over ``g77``, but if you happen to have both
+installed then ``g77`` will be detected and used first.  To explicitly select
+``gfortran`` in that case, do::
+
+    python setup.py build --fcompiler=gnu95
+
+Windows
+-------
+
+On Windows, building from source can be difficult.  Currently the most robust
+option is to use the Intel compilers, or alternatively MSVC (the same version
+as used to build Python itself) with Intel ifort.  Intel itself maintains a
```

Comment:
No, will make that clear. Not needed for Numpy itself (``f2py`` tests will use a Fortran compiler if it's available, but that's it), but important for building Scipy.